dataset/table.py

```diff
@@ -31,9 +31,8 @@ def __init__(self, name, src, *args):
                 # Normalize to lower strings and strip whitespaces.
                 # TODO: No support for numerical values. To be added.
                 for attr in self.df.columns.values:
-                    if attr != '_tid_' and self.df[attr].dtype == str:
-                        self.df[attr] = self.df[attr].apply(lambda x: x.lower())
-                        self.df[attr] = self.df[attr].apply(lambda x: x.strip())
+                    if attr != '_tid_':
+                        self.df[attr] = self.df[attr].apply(lambda x: x.lower().strip() if type(x) == str else x)
```

Comment:
might conflict with https://github.com/HoloClean/holoclean/pull/18/files?utf8=%E2%9C%93&diff=split&w=1#diff-c25c4677d626699f60b8cf3ab8235f4dR51
Use the other PR as a guidance to change this if conflicts